scripts/post-release-notes-preview.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/local/bin/python3
+
+import os
+import requests
+import subprocess
+import sys
+
+BRANCH_PREFIXES_TO_POST = ['feat', 'fix', 'chore', 'test', 'docs', 'revert']
+GIT_BRANCH = os.environ['CIRCLE_BRANCH']
+gitBranchParts = GIT_BRANCH.split('/')
+if len(gitBranchParts) <= 1:
+    sys.exit(0)
+
+gitBranchPrefix = gitBranchParts[0]
+if not gitBranchPrefix in BRANCH_PREFIXES_TO_POST:
+    sys.exit(0)
+
+SLACK_WEBHOOK = os.environ['SLACK_WEBHOOK']
+USERNAME = os.environ.get('CIRCLE_USERNAME', 'USER_NOT_FOUND')
```

Comment:
Should not having a user ever happen?

Reply:
when running locally :)
or... CircleCI freaking out?